tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_sync_committee_updates.py

```diff
@@ -17,11 +18,20 @@
 )
 
 
+#
+# Note:
+# Calculating sync committees requires pubkey aggregation, thus all tests are generated with `always_bls`
+#
+
```

Comment:
do you want a spare line here? 

```suggestion
```

Reply:
yes, I think so as it's a header comment for the whole file